### DISCUSSION TITLE
Make UI (Song select) background blur a slider instead of a toggle like Gameplay blur

### DISCUSSION THREAD

**Discussion by crxnier** [ppy/osu#26900] [Ideas]
![image](https://github.com/ppy/osu/assets/37815721/317a9b55-6f55-4c07-b9e4-55667d314e09)

title say it all mostly but I personally would like some background blur but find the current one too much and having a slider would allow for more customisability

**Comment by VikrorLion** [comment]
Thnks)

**Comment by OwenCMYK** [comment]
I firmly agree with this. Honestly a dim might be cool too just like gameplay.
